src/lib.rs

```diff
@@ -59,12 +59,14 @@ impl Bencher {
     ///
     /// # Timing loop
     ///
-    /// ``` ignore
-    /// start = Instant::new();
+    /// ```rust,no_run
+    /// # fn routine() {}
+    /// let iters = 4_000_000;
+    /// let start = std::time::Instant::now();
     /// for _ in 0..iters {
     ///     routine();
     /// }
-    /// elapsed = start.elapsed();
+    /// println!("{:?}", start.elapsed());
```

Comment:
revert this back to the `elapsed` binding because the "Timing model" section below refers to it.